frontend/src/components/Header.jsx

```diff
@@ -0,0 +1,24 @@
+import React, {useState} from "react";
+import Navigation from "./Navigation";
+import {ReactComponent as Logo} from "../library_logo.svg";
+import './header.scss';
+
+export default function Header() {
+    const [navOpen, setNavOpen] = useState(false);
+
+    const toggleNav = () => setNavOpen(!navOpen);
```

Comment:
Here you use a function for only one line of code, you cand do this directly in the code, where you use the function. It just spares code^^